src/app/Navbar.tsx

```diff
@@ -0,0 +1,52 @@
+"use client"
+
+import Image from "next/image"
+import SolvroLogo from "./public/solvro_dark.png"
+import PromochatorLogo from "./public/promochator_logo.png"
+
+import {useState} from "react"
+
+export default function () {
+  const [showSidebar, setShowSidebar] = useState<boolean>(false)
+  return (
+    <>
+      <div className="w-full flex flex-row justify-between items-center gap-12">
+        <div className="flex flex-row gap-2 items-center mr-auto">
+          <Image src={PromochatorLogo} alt="Promochator Logo" className="w-6"/>
+          <p className="text-lg font-semibold">Promo<span className="text-[#5f7ecd]">CHAT</span>or</p>
+        </div>
+        <div className="hidden lg:flex flex-row gap-12 whitespace-nowrap">
+          <p>Jak to działa?</p>
+          <p>Jak ci pomożemy?</p>
+          <p>Regulamin</p>
+        </div>
+        <Image src={SolvroLogo} alt="Solvro Logo" className="hidden lg:flex w-40 ml-auto"/>
+        {/* Mobile icon */}
+        <button title="Otwórz menu" onClick={() => setShowSidebar(true)}
+                className="w-6 h-6 flex lg:hidden flex-col gap-2 items-center justify-center">
+          <span className="block bg-white w-6 h-0.5"/>
+          <span className="block bg-white w-6 h-0.5"/>
```

Comment:
Imo, we can use Lucide icons there
https://lucide.dev/guide/packages/lucide-react
It's already installed